test_main.py

```diff
@@ -1,4 +1,7 @@
 import unittest2
+from bootstrap import bootstrap
+
+bootstrap()
```

Comment:
Add a comment for posterity on why this is needed.

Reply:
Fixed